app/views/course_management/enrolled_courses/index.html.erb

```diff
@@ -0,0 +1,75 @@
+<div class='row'>
+  <div class='col-lg-12'>
+    <div class="alert alert-light">
+      <%= link_to_back root_path %>
+    </div>
+
+    <div class="card">
+      <div class="card-header">
+        <%= fa_icon 'align-justify' %><strong><%= t('.student_records') %></strong>
+      </div>
+      <div class="card-body">
+        <div class="list-group list-group-sm">
+          <% @students.each do |student| %>
+            <%= link_to enrolled_courses_url(student_id: student.id), class: "list-group-item list-group-item-action justify-content-between align-items-center" do %>
+              <%= student.unit.name %>
+            <% end %>
+          <% end %>
+        </div>
+      </div>
+    </div>
+
+    <div class="card">
+      <div class="card-header">
+        <%= fa_icon 'user' %><strong><%= t('.student_info') %></strong>
+      </div>
+      <div class="card-body table-responsive">
+        <table class='table table-bordered table-hover'>
+          <tbody>
+            <tr>
+              <th><%= t('.student_number') %></th>
+              <td><%= @student.student_number %></td>
+            </tr>
+            <tr>
+              <th><%= t('.program') %></th>
+              <td><%= @student.unit.name %></td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+  </div>
+</div>
+
+<% if @student.registrable_for_online_course? %>
+  <div class='row'>
+    <div class="col-sm-6">
+      <div class="card border-default mb-3">
+        <div class="card-header">
+          <strong><%= t('.available_courses') %></strong>
+        </div>
+        <% @semesters.each do |semester| %>
+          <div class="card-body">
+            <%= render 'semesters', semester: semester %>
+          </div>
+        <% end %>
+      </div>
+    </div>
+    <div class="col-sm-6">
+      <div class="card border-success mb-3">
+        <div class="card-header">
+          <strong><%= t('.selected_courses') %></strong>
```

Comment:
Ders seçimi burada yapılacaksa, alacağı derslerin görüntülenmesi sayfası için `enrolled_courses` yerine `course_registrations` yolu kullanılabilir.  Bu isimlendirme konusunda pek iyi değilimdir. Sadece sesli düşünüyorum.

Reply:
Haklısın @ecmelkytz , "Ders Kaydı" ekranı için `enrolled_courses` pek iç açıcı durmuyor. Ama controller adından mı bahsediyordun, route'dan mı? "Kayıt olunmuş dersler" modelinin yönetimi gibi düşünmüştüm, düzeni bozmamak adına `enrolled_courses_controller` dedim, sonrasında bir değişiklik olursa değiştiririz diye. Şimdi de değiştirebiliriz.

Reply:
Ben ekranda olanlara baktığımda ders seçiminin de orada yapılacağını anladığım için controller'dan itibaren değiştirilebileceğini düşündüm. Ama bu işlemi #1009'da yapacağın için geliştirme aşamasında duruma göre değişiklik yapabilirsin.

Reply:
@dilara Ecmel'e bu konuda katılıyorum. Diğer iş kaydını yaparken senden birkaç şey isteyeceğim.

- Bu controller ismini `CourseEnrolmentsController` yapalım. (Eğer isimde herkes hem fikir olursa)
- Bu kontrolürü `CourseManagement` modülünün altından, `Studentship` modülü altına alalım (`Studentship` modülü yok oluşturalım), çünkü bu yapılan işler artık öğrencilikle ilgili olan işler. (**Not:** Modüle ismini `Student` yerine `Studentship` yazdım çünkü isim çakışması yaşanır. Eğer isimde herkes hem fikir olursa bu şekilde devam edelim.)
- `CourseEnrolmentsController`'nın `index` sayfasına ders kayıtlanma tarihleri, ders kayıtlanmasına engel olan durumlar hakkında bilgilendirmeler ve yapılan ders kaydının son durumunu koyalım. Bu sayfa üzerinden ders kayıtlanma sayfasına ulaşsın. `CourseEnrolmentsController`'nın `new' işlevinde ise ders kayıtlanma formunu görüntüleyelim.

Reply:
Yazdıklarımı bu PR için yapma, diğer PR'lar için ön hazırlık ve beyin fırtınası olması adına yazdım.

Bu PR içinde eline sağlık :+1:

Reply:
> Yazdıklarımı bu PR için yapma, diğer PR'lar için ön hazırlık ve beyin fırtınası olması adına yazdım.

O vakit ortada bir `debt` var.  @dilara ellerine sağlık! :+1:  @isubas ve @ecmelkytz'nin review'lerinden hareketle gelecekte yapılması gereken düzeltmeleri kapsayan `debt` etiketli bir iş kaydı açalım ve bu PR'ın açıklamasına `debt` kaydı olarak ekleyelim.  PR şablon metninde bu zaten vardı, şöyle bir şey (copy/paste yapabilirsin):

```markdown
#### Teknik borç kayıtları

[//]: # (PR ile geliştirilen çözümün eksiklikleri varsa "Teknik borç" türünde iş kayıtları oluşturulmalı)
[//]: # (İş kaydını ön izlemeye al ve aşağıda görüntülenen bağlantıyı yeni sekmede aç)
[//]: # (Bu şekilde oluşturulan kayıtları `References` anahtar kelimesiyle listele)
[//]: # (DİKKAT! TEKNİK BORÇ YOKSA AŞAĞIDAKİ SATIRI N/A İLE DEĞİŞTİRİN VEYA SİLİN!)

[Teknik borç](../issues/new?labels=debt&template=debt.md)
```

Reply:
@dilara bu `debt` konusunu çok dokümante edemedik, şimdilik şu kadar bir [açıklama](https://doc.omu.sh/workflow/label/):

>  Geliştiriciler debt etiketini her PR'da serbest şekilde kullanabilir. Bu etiket zamanında tamamlanması gereken bir iş kaydını kapatabilecek yeterlilikte açılan bir PR'da yetiştirilemeyen veya eksik bırakılan işleri kayıt altına almak için kullanılır.

> debt etiketli iş kayıtları sınırsız artmamalıdır. Sayının çok artması halinde proje yöneticileri belirli debt kayıtlarını urgent olarak etiketleyerek ilgili tarafları hemen bir çözüm geliştirmeye zorlayabilir. Bu amaçla debt, urgent ve discuss etiketleri birlikte kullanılabilir.